doc/manual_rest.rst

```diff
@@ -418,6 +418,7 @@ instead of the default, set the environment variable like this:
     $ restic -r /srv/restic-repo backup ~/work
 
 
+If the environment variable ``$RESTIC_CACHE_DIR`` is not set, ``check`` creates its temporary cache directory in the temporary directory. If ``$RESTIC_CACHE_DIR`` is set, ``check`` creates its temporary cache directory in ``$RESTIC_CACHE_DIR``.
```

Comment:
I think that is the wrong place to describe the behavior of `check` in regards to the cache directory location. Either integrate the explanation in the `Caching` section below, or explain the caching behavior together with the general description of the `check` command in https://restic.readthedocs.io/en/stable/045_working_with_repos.html#checking-integrity-and-consistency .

P.S.: As described below, the `--cache-dir` can also be used to change the location of the cache directory.